recipes/arrow/all/patches/1.0.0-0001-cmake.patch

```diff
@@ -66,28 +33,22 @@
  
    # TODO: Don't use global includes but rather target_include_directories
 -  get_target_property(SNAPPY_INCLUDE_DIRS Snappy::snappy INTERFACE_INCLUDE_DIRECTORIES)
-+  get_target_property(SNAPPY_INCLUDE_DIRS Snappy::Snappy INTERFACE_INCLUDE_DIRECTORIES)
-   include_directories(SYSTEM ${SNAPPY_INCLUDE_DIRS})
+-  include_directories(SYSTEM ${SNAPPY_INCLUDE_DIRS})
++  #get_target_property(SNAPPY_INCLUDE_DIRS Snappy::snappy INTERFACE_INCLUDE_DIRECTORIES)
```

Comment:
any idea why `get_target_property` doesn't work?

Reply:
When with_snappy is False (default), `SNAPPY_INCLUDE_DIRS` returns /usr/include and `CONAN_INCLUDE_DIRS_SNAPPY` returns nothing. When `with_snappy` is True, and Snappy is found, both returns the same correct include path. Thus, get_target_property should work.

Reply:
I can not remember which PR it was in but https://github.com/conan-io/conan/blob/0c8c454e78fefee05eaf0089911ae3d280ce80c1/conans/client/generators/cmake_find_package.py#L36

would be generated with the wrong casing, and it's know to cause problems... 

it was https://github.com/conan-io/conan/pull/7320 or https://github.com/conan-io/conan/pull/7382

Reply:
@prince-chrismc thanks, Yes, unfortunately we can't manipulate the variable yet. Not it makes sense

Reply:
`conan create . arrow/1.0.0@ -o arrow:with_snappy=True` works without the change.
I don't remember why I did this change.
Probably some cmake mocules clashing and/or the snappy conan cmake name not correct yet.